### ISSUE TITLE
Passed props bleed to subsequent instances

### ISSUE THREAD

**Issue by trurl-master** [zouhir/preact-habitat#24]
```javascript

<div class="my-component">
    <script type="text/props">
        {"a": 1, "b": 2}
    </script>
</div>

<div class="my-component">
    <script type="text/props">
        {"a": 3}
    </script>
</div>


habitat(MyComponent).render({
    selector: '.my-component'
});
```
The second instance will receive `{"a": 3, "b": 2}`

If I understand correctly it is caused by directly mutating props in `collectPropsFromElement`

**Comment by zouhir** [comment]
oh shit.. I think you are right, that's a MUST fix real asap...

**Comment by zouhir** [comment]
I'll look at it next week, if you have time & can help happy to merge it earlier <3

**Comment by sampotts** [comment]
Did this PR get released? Seeing this myself at the moment...

**Comment by zouhir** [comment]
Releasing in the next few hours @sampotts!

**Comment by zouhir** [comment]
I've just released 3.2.2 <3 thanks a lot for your patience!
